<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LazyQuadtreeLodPlot</class>
 <widget class="QWidget" name="LazyQuadtreeLodPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="plotFrame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>281</width>
     <height>151</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="maxRecursion">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>49</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>24</number>
   </property>
   <property name="value">
    <number>12</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="lodFactor">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="decimals">
    <number>9</number>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.000100000000000</double>
   </property>
   <property name="value">
    <double>0.000100000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
